vcd/resource_vcd_vapp.go

```diff
@@ -249,6 +255,26 @@ func resourceVcdVAppCreate(d *schema.ResourceData, meta interface{}) error {
 		}
 	}
 
+	if _, ok := d.GetOk("properties"); ok {
+		// TODO - this is just for testing. Real issue must be solved
```

Comment:
what is this ?why we need sleep?

Reply:
Just as per the comment above :) This is a `TODO` leftover

Reply:
This is not removed and a solution of `err = vapp.BlockWhileStatus("UNRESOLVED", vcdClient.MaxRetryTimeout)` is put into place